CHANGELOG.md

```diff
@@ -1,10 +1,13 @@
 # Changelog
 
+## 0.21.1 (unreleased)
+- [[#129](https://github.com/IronCoreLabs/ironoxide/pull/129)] Improved error message for SDK initialization failure
```

Comment:
I think that even with a single entry, we've tended to put the PR and the description on two separate lines